alivee/utils/requests/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class RequestMethodNotSupportedException(Exception):
+    pass
```

Comment:
When we log Exceptions or throw exceptions, python has this low level traceback. For example:

```
def f():
    d()

def d():
    raise Exception

f()
```

Traceback (most recent call last):
  File "/Users/Winton/Desktop/temp.py", line 7, in <module>
    f()
  File "/Users/Winton/Desktop/temp.py", line 2, in f
    d()
  File "/Users/Winton/Desktop/temp.py", line 5, in d
    raise Exception
Exception